infrastructure/zk/src/docker.ts

```diff
@@ -46,8 +45,12 @@ async function _build(image: string) {
     }
     const { stdout: imageTag } = await utils.exec('git rev-parse --short HEAD');
     const latestImage = `-t matterlabs/${image}:latest`;
-    const taggedImage = ['nginx', 'server', 'prover'].includes(image) ? `-t matterlabs/${image}:${imageTag}` : '';
-    await utils.spawn(`DOCKER_BUILDKIT=1 docker build ${latestImage} ${taggedImage} -f ./docker/${image}/Dockerfile .`);
+    const taggedImage = ['nginx', 'server', 'prover', 'dev-liquidity-token-watcher'].includes(image)
```

Comment:
Why is this necessary? Why do we want to tag it differently? I'd change it once we manage to publish the images under `rsksmart`

Reply:
removed